tests/python_package_test/test_arrow.py

```diff
@@ -38,13 +37,13 @@ def generate_dummy_arrow_table() -> pa.Table:
     return pa.Table.from_arrays([col1, col2], names=["a", "b"])
 
 
-def generate_random_arrow_table(num_columns: int, num_datapoints: int, seed: int) -> pa.Table:
+def generate_random_arrow_table(num_columns, num_datapoints, seed) -> pa.Table:
```

Comment:
@borchero I might not have been clear in https://github.com/microsoft/LightGBM/pull/6166#discussion_r1392709299.

Please preserve the type hints for arguments and returns on utility functions referenced from tests... they're helpful documentation of how data is flowing through the tests.

I just was asking that you remove type hints for `pytest` test cases (any function whose name begins with `test_`).

Reply:
Whoops, sure :smile: